pages/summary.js

```diff
@@ -0,0 +1,46 @@
+// node_modules
+
+import PropTypes from "prop-types";
+import { API_URL } from "../lib/constants";
+import Breadcrumbs from "../components/breadcrumbs";
+
+export default function Page({ data }) {
+  return (
+    <>
+      <Breadcrumbs />
+      <h1>Summary</h1>
+      <div dangerouslySetInnerHTML={{ __html: data }} />
+    </>
+  );
+}
+
+Page.propTypes = {
+  data: PropTypes.string.isRequired,
+};
+
+export async function getServerSideProps({ query }) {
+  const queryElements = [];
+  for (const [key, value] of Object.entries(query)) {
+    const element = key + "=" + value;
+    queryElements.push(element);
+  }
+  const queryString = queryElements.join("&");
```

Comment:
`getQueryStringFromServerQuery()` does this composition of a query string from the server query. The code here won’t work right if you have repeated query-string keys (e.g. "type=AnalysisSet&type=MeasurementSet") because the keys appear only once in `query` but its value is an array of values, which won’t work right here. This also applies to search.js, though this code isn’t in the diffs in that case.